README.md

```diff
@@ -51,6 +51,8 @@ Options:
                         repeatable.
   --force-color         Try force colored output (relying on ansible's code)
   --nocolor             disable colored output
+  -F F                  Specify configuration file to use.  Defaults to
```

Comment:
Can you make the second F here /path/to/file (or similar if /path/to/file won't fit)